### ISSUE TITLE
Set x/y position for SVG draw

### ISSUE THREAD

**Issue by Ischtasu** [svg-net/SVG#913]
#### Description

The SVG graphic is always positioned at point 0, 0. The transfer of the values is ignored when drawing. The assigned values (e.g. x = 25; y = 125) are not transferred to the SVG object.

In version 3.0.102 the values were taken over correctly and the SVG graphics were positioned.
In this version Integer values could be used directly. The conversion to Single and then to SvgUnit was an attempt to adapt it to the new version. Unfortunately, this failed.

#### Example data
```
Dim recoloredSVG As SvgDocument = SvgDocument.Open(DocSVG.BaseUri.LocalPath)
Dim x As Integer = 25
Dim y as Integer = 125
Dim x As Single = Convert.ToSingle(x_icon)
Dim y As Single = Convert.ToSingle(y_icon)

Dim svg_x As SvgUnit = New SvgUnit(SvgUnitType.Pixel, x)
Dim svg_y As SvgUnit = New SvgUnit(SvgUnitType.Pixel, y)

With recoloredSVG
	.Width = w_icon
	.Height = h_icon
	.X = svg_x
	.Y = svg_y
	.Draw(e.Graphics)
End With
```

#### Used Versions

Nuget Version 3.3.0
.NET Framework 4.8
OS Win 10 21H1 (Build 19043.1288)

**Comment by faiith** [comment]
yeah, can confirm the issue. In version 3.2.3 all is working fine, but 3.3.0 is broken

**Comment by H1Gdev** [comment]
#886 and #887.

**Comment by H1Gdev** [comment]
@Ischtasu

As mentioned in [PR](https://github.com/svg-net/SVG/pull/887#issue-981716196), we merged PR #887 that conforms to SVG specification.

You can do the same by doing the following.
```cs
var size = System.Drawing.Size.Round(svgDoc.GetDimensions());
var bitmap = new System.Drawing.Bitmap(size.Width, size.Height);
var renderer = SvgRenderer.FromImage(bitmap);
renderer.TranslateTransform(100f, 100f);
svgDoc.Draw(renderer);
```
Please dispose instances(Bitmap, SvgRenderer) as appropriate.

**Comment by mrbean-bremen** [comment]
So, please confirm: this is not a bug, but a change to conform to the SVG standard (as I now remember), but it changed the behavior, so existing code is affected.
We should at least have made this more clear in the release notes, not sure if we shall change something in te code.
@H1Gdev - what do you think?
